OOPLB3/Read/CommandFile.cpp

```diff
@@ -0,0 +1,83 @@
+//
+// Created by akimfeopentov on 04.11.22.
+//
+#include <algorithm>
+#include "CommandFile.h"
+
+CommandFile::CommandFile() {
+    file.open("Command.txt");
+    strType.push_back("down");
+    strType.push_back("up");
+    strType.push_back("left");
+    strType.push_back("right");
+    strType.push_back("esc");
+}
+
+bool CommandFile::get_map(std::map<char, CommandType> &map) {
+    while (!file.eof()) {
+        std::string str;
+        std::getline(file, str);
+        std::transform(str.begin(), str.end(), str.begin(), tolower);
+        delete_space(str);
+        prov_string(str);
+    }
+    if (mapLocal.size() == 5) {
+        map = mapLocal;
+        return true;
+    } else {
+        map['s'] = CommandType::DOWN;
+        map['w'] = CommandType::UP;
+        map['a'] = CommandType::LEFT;
+        map['d'] = CommandType::RIGHT;
+        map['l'] = CommandType::ESC;
+        return false;
+    }
+
+}
+
+void CommandFile::delete_space(std::string &str) {
+    for (int i = 0; i < str.length(); i++) {
+        if (str[i] == ' ') {
+            str.erase(i, 1);
+            i--;
+        }
+    }
+}
+
+void CommandFile::prov_string(std::string str) {
```

Comment:
```suggestion
void CommandFile::parse_string(std::string str) {
```
Чисто как предложение